sdcm/tester.py

```diff
@@ -1866,19 +1867,20 @@ def run_stress_cassandra_thread(
             self.update_stress_cmd_details(stress_cmd, prefix, stresser="cassandra-stress",
                                            aggregate=stats_aggregate_cmds)
         stop_test_on_failure = False if not self.params.get("stop_test_on_stress_failure") else stop_test_on_failure
-        cs_thread = CassandraStressThread(loader_set=self.loaders,
-                                          stress_cmd=stress_cmd,
-                                          timeout=timeout,
-                                          stress_num=stress_num,
-                                          keyspace_num=keyspace_num,
-                                          compaction_strategy=compaction_strategy,
-                                          profile=profile,
-                                          node_list=self.db_cluster.nodes,
-                                          round_robin=round_robin,
-                                          client_encrypt=self.params.get('client_encrypt'),
-                                          keyspace_name=keyspace_name,
-                                          stop_test_on_failure=stop_test_on_failure,
-                                          params=params or self.params).run()
+        cls = CqlStressCassandraStressThread if "cql-stress" in stress_cmd else CassandraStressThread
```

Comment:
I would rather have this explicit in its own function

This is a bit confusing to follow

Reply:
Agree.
Moreover, `cls` is reserved name for the current class reference.